source/reference/minio-mc-admin/mc-admin-config.rst

```diff
@@ -278,8 +293,44 @@ HTTP Webhook Audit Log Target
 
       Requires specifying :mc-conf:`~audit_webhook.client_cert`.
 
-      This setting corresponds to the
-      :envvar:`MINIO_AUDIT_WEBHOOK_CLIENT_KEY` environment variable.
+      This setting corresponds to the :envvar:`MINIO_AUDIT_WEBHOOK_CLIENT_KEY` environment variable.
+
+   .. mc-conf:: queue_dir
+
+      .. versionadded:: RELEASE.2023-05-18T00-05-36Z
+
+      *Optional*
+
+      Specify the directory path to enable MinIO's persistent event store for undelivered messages, such as ``/home/events``.
+
+      MinIO stores undelivered events in the specified store while the webhook service is offline and replays the stored events when connectivity resumes.
+
+      This setting corresponds to the :envvar:`MINIO_AUDIT_WEBHOOK_QUEUE_DIR` environment variable.
+
+   .. mc-conf:: kafka_queue_dir
```

Comment:
I made up this name because I don't know what the corresponding command is for `MINIO_AUDIT_KAFKA_QUEUE_DIR`

Answer TBD

Reply:
```
mc admin config get play audit_kafka                                                                       ✔  docs   system   
audit_kafka enable=off topic= brokers= sasl_username= sasl_password= sasl_mechanism=plain client_tls_cert= client_tls_key= tls_client_auth=0 sasl=off tls=off tls_skip_verify=off version= queue_size=100000 queue_dir=
```

There are ways to find this in code too, but this is far easier. 

So this actually needs the root `.. mc-conf:: audit_kafka` key first.

I think I didn't get around to documenting the kafka stuff - we can drop that for now and create a new issue for kafka audit, which I'll take responsibility for - just to unblock this.

Reply:
Here's the new issue
https://github.com/minio/docs/issues/886

Let's discuss next week. If I sufficiently understand what is needed, no reason for you to take this on.